java.d/src/main/java/org/firehol/netdata/utils/LoggingUtils.java

```diff
@@ -22,7 +22,12 @@
 import java.io.StringWriter;
 import java.util.function.Supplier;
 
-public abstract class LoggingUtils {
+import lombok.experimental.UtilityClass;
```

Comment:
I am not sure if we should use Lombok experimatal features for easy things like this.